src/FSharpVSPowerTools.Logic/VSUtils.fs

```diff
@@ -58,16 +53,16 @@ let isPhysicalFileKind (kind: string) =
     kind.Equals(EnvDTE.Constants.vsProjectItemKindPhysicalFile, StringComparison.OrdinalIgnoreCase)
 
 let isPhysicalFileOrFolderKind kind =
-    kind <> null && (isPhysicalFolderKind kind) || (isPhysicalFileKind kind)
+    isNotNull kind && (isPhysicalFolderKind kind) || (isPhysicalFileKind kind)
```

Comment:
I don't think replacing all `= null` and `<> null` with `isNull` / `isNotNull` makes code better. These parts of code are called not very frequently and worse `=` `<>` performance should not drive code style here.